dnf-behave-tests/dnf/repo-packages-upgrade.feature

```diff
@@ -47,3 +47,24 @@ Given I use repository "dnf-ci-fedora-updates"
       | upgrade       | glibc-0:2.28-26.fc29.x86_64               |
       | upgrade       | glibc-common-0:2.28-26.fc29.x86_64        |
       | upgrade       | glibc-all-langpacks-0:2.28-26.fc29.x86_64 |
+
+
+Scenario: repository-packages upgrade doesn't reinstall a pkg if it has same nevra but different vendor or build-time
+Given I use repository "simple-base"
+  And I execute dnf with args "install vagare"
```

Comment:
I am quite curious why did you create `vagare` package because it does not ave any special feature? (it is not an issue)

Reply:
I needed two nearly identical `vagare` packages. One is in `simple-base` and here I am creating the second one which has a different `Vendor` but the same nevra. The test is testing whether we reinstall the package just because of that.
It is basically testing: https://github.com/openSUSE/libsolv/issues/287

I will add a comment to the test to explain it.

Reply:
OK, thank you for an explanation